src/styles/utility/mixins.scss

```
@@ -0,0 +1,9 @@
+@mixin star-size($star-size) {
+  height: $star-size;
+  width: $star-size;
+}
+
+@mixin logo-size($logo-size) {
+  height: $logo-size;
+  width: $logo-size;
+}
```

Use one mixin for all elements
```suggestion
@mixin square($size) {
  height: $size;
  width: $size;
}
```